pkg/scalers/scaler.go

```diff
@@ -44,7 +43,7 @@ func init() {
 type Scaler interface {
 
 	// The scaler returns the metric values for a metric Name and criteria matching the selector
-	GetMetrics(ctx context.Context, metricName string, metricSelector labels.Selector) ([]external_metrics.ExternalMetricValue, error)
+	GetMetrics(ctx context.Context, metricName string) ([]external_metrics.ExternalMetricValue, error)
```

Comment:
This change was missed in the docs - https://github.com/kedacore/keda-docs/blob/main/content/docs/2.9/concepts/external-scalers.md

Reply:
Good catch, thanks!